clickhouse/base/wire_format.cpp

```diff
@@ -0,0 +1,105 @@
+#include "wire_format.h"
+
+#include "input.h"
+#include "output.h"
+
+#include <stdexcept>
+
+namespace {
+static const int MAX_VARINT_BYTES = 10;
+}
```

Comment:
```suggestion
namespace {
constexpr int MAX_VARINT_BYTES = 10;
}
```
No need in static with unnamed namespaces.